modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/StandardHeader.scala

```diff
@@ -199,8 +199,8 @@ case class StateKeywordsReader(conditions: Conditions, operator: Option[Operator
   def encodeCondition(c: Int): String = if(c == 100) "Any" else s"$c-percentile"
 
   // TODO: "observer" should be the default when not set in state
-  def getObserverName: SeqAction[String] = SeqAction(observer.getOrElse("observer"))
-  def getOperatorName: SeqAction[String] = SeqAction(operator.getOrElse("ssa"))
+  def getObserverName: SeqAction[String] = SeqAction(observer.filter(_.nonEmpty).getOrElse("observer"))
+  def getOperatorName: SeqAction[String] = SeqAction(operator.filter(_.nonEmpty).getOrElse("ssa"))
```

Comment:
I guess these are the default keyword values when no operator or observer are defined, is that right?